### ISSUE TITLE
Order of location bar suggestions

### ISSUE THREAD

**Issue by Gitoffthelawn** [Quicksaver/The-Fox--Only-Better#191]
When the number of location bar suggestions is increased, the order of them is peculiar.

Often, the first item is a search item, the next 5 items are bookmarks, then a group of search suggestions, then more bookmarks.

Can you make it so all the bookmarks are grouped together, preferably before the larger set of search suggestions?

Beyond Australis Extension version 1.4.6
Firefox version 48.0.2

**Comment by Quicksaver** [comment]
The order those suggestions are shown are defined by Firefox, using an internal score for each suggestion. This score is based on the number of times you've visited that page, the last time you visited it, and a few other factors (I don't know the exact formula to be honest as I never investigated, perhaps it's documented somewhere).

I don't plan on changing this at the moment to be honest as there are workarounds. See [this page](https://support.mozilla.org/kb/awesome-bar-search-firefox-bookmarks-history-tabs#w_changing-results-on-the-fly) for a way to search only in your history or in your bookmarks for instance.

Also, you should definitely update your Firefox. While Firefox 48 has a couple of incompatibilities with Beyond Australis, which I plan to fix hopefully soon, I believe it's sorting mechanism of suggestions has much improved since 38. And version 1.4.6 of the add-on isn't really compatible with that version, it needs at least Firefox 45 to work correctly.

**Comment by Gitoffthelawn** [comment]
Thanks!  I will take a close look at it.  I think the increased number of suggestion items available when using *Beyond Australis* makes the use of Firefox's *frecency* algorithm less desirable.  I'll play with it some more.

Regarding the version of Firefox, I agree completely.  I also apologize for the time you spent writing that last paragraph (although it is good information!).  I butterfingered, and typed a "3" when I meant to type "4" (I now fixed the typo).  I also think mentally, I can't believe we are already at version 48 and that Mozilla actually adopted that silly version numbering system! :smile:
